src/components/Presentation/index.jsx

```diff
@@ -31,8 +33,26 @@ class Presentation extends React.Component {
   addSlide = () => {
     this.props.dispatch({type: 'CREATE_SLIDE', payload: {title: 'Title', subtitle: 'Subtitle'}});
   }
+  updateCanvas() {
+    let canvas = new fabric.Canvas(this.canvas.current);
```

Comment:
I think we might need to do some refactoring on this later on, as it's currently going to create a new canvas every time we create a slide. Ideally we would want to avoid doing that because viewing other slide data could be very cumbersome.